modules/nf-core/samshee/tests/main.nf.test

```diff
@@ -0,0 +1,85 @@
+nextflow_process {
+
+    name "Test Process samshee"
+    script "../main.nf"
+    process "SAMSHEE"
+    config "./nextflow.config"
+    tag "modules"
+    tag "modules_nfcore"
+    tag "samshee"
+
+    test("test samplesheet_v1") {
+
+        when {
+            params {
+                v1_schema             = true
+                json_schema_validator = '{"required": ["Data"]}'
+                name_schema_validator = null
+            }
```

Comment:
I normally pass these via `ext.args` and using different configs, but this works anyway 👍 .